lifecycle_msgs/package.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>lifecycle_msgs</name>
+  <version>0.0.0</version>
+  <description>A package containing some lifecycle related message and service definitions.</description>
+  <maintainer email="karsten@osrfoundation.org">Karsten Knese</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>rosidl_default_generators</buildtool_depend>
+
+  <exec_depend>rosidl_default_runtime</exec_depend>
+
+  <test_depend>ament_lint_common</test_depend>
```

Comment:
The lines is needed for `ADD_LINTER_TESTS`. If you want to also lint CMake etc. you might also want to depend on `ament_lint_auto` and use the CMake command to run all linters (https://github.com/ros2/system_tests/blob/0a2daef45486c93efb42c92e7c3024cc403847e4/test_communication/CMakeLists.txt#L16-L17).